services/clamav/src/index.js

```diff
@@ -1,28 +1,45 @@
 /* istanbul ignore file */
-const NodeClam = require("clamscan");
+const axios = require("axios").default;
+const FormData = require("form-data");
+
 const fs = require("fs");
+const path = require("path");
+
+const { CLAM_AV_HOST } = process.env;
+
+module.exports = async (fileInformation, { fileName, location, debug }) => {
+  const host = CLAM_AV_HOST || "https://dev-clamav.azurewebsites.net";
```

Comment:
I'm assuming this is an additional requirement for infrastructure? Where is this captured?

Reply:
As discussed, there is no changes in terms of infrastructure to be documented in confluence.

Reply:
No changes to be documented, but it will require a change to the terraform work that is being done by Ravi. Have you ensured that this has been captured in a JIRA ticket?

Reply:
It hans't been captured but will action this.